---
title: "Pan-cancer proteomic map of 949 human cell lines reveals principles of cancer vulnerabilities"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- admin 
- Emanuel Gon√ßalves
- Rebecca C Poulos
- Syd Barthorpe
- Srikanth S Manda
- Natasha Lucas
- Alexandra Beck
- Daniel Bucio-Noble
- Michael Dausmann
- Caitlin Hall
- Michael Hecker
- Jennifer Koh
- Sadia Mahboob
- Iman Mali
- James Morris
- Laura Richardson
- Akila J Seneviratne
- Erin Sykes
- Frances Thomas
- Sara Valentini
- Steven G Williams
- Yangxiu Wu
- Dylan Xavier
- Karen L MacKenzie
- Peter G Hains
- Brett Tully
- Phillip J Robinson
- Qing Zhong
- Mathew J Garnett
- Roger R Reddel

date: "2022-07-14T00:00:00Z"
doi: "https://doi.org/10.1016/j.ccell.2022.06.010"

# Schedule page publish date (NOT publication's date).
publishDate: "2017-07-14T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: In *Cancer Cell*
publication_short: In *Cancer Cell*

abstract: The proteome provides unique insights into disease biology beyond the genome and transcriptome. A lack of large proteomic datasets has restricted the identification of new cancer biomarkers. Here, proteomes of 949 cancer cell lines across 28 tissue types are analyzed by mass spectrometry. Deploying a workflow to quantify 8,498 proteins, these data capture evidence of cell-type and post-transcriptional modifications. Integrating multi-omics, drug response, and CRISPR-Cas9 gene essentiality screens with a deep learning-based pipeline reveals thousands of protein biomarkers of cancer vulnerabilities that are not significant at the transcript level. The power of the proteome to predict drug response is very similar to that of the transcriptome. Further, random downsampling to only 1,500 proteins has limited impact on predictive power, consistent with protein networks being highly connected and co-regulated. This pan-cancer proteomic map (ProCan-DepMapSanger) is a comprehensive resource available at https://cellmodelpassports.sanger.ac.uk.


tags: ["Deep Learning", "Machine learning", "Multi-omics", "Pan-cancer", "Drug response", "CRISPR-Cas9"]

# Display this page in the Featured widget?
featured: true

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
projects:
- sanger2020

events: 
- kca2021
- abacbs2021sanger
---

