---
title: A pan-cancer proteomic map of 949 human cell lines

event: THE COMBINED ABACBS AND PHYLOMANIA 2021 HYBRID CONFERENCE
event_url: https://www.abacbs.org/conference2021

location: Virtual/Australia

abstract: "Omics data analysis, powered by machine learning, has significantly improved cancer diagnosis and prognosis. However, most machine learning methods consider each gene as an independent feature, failing to integrate experimentally-acquired gene regulation and pathway information. The benefit of utilising this information increases in the era of multi-omics, because gene regulation is the key mechanism that links different omic layers together. Here, we present an interpretable deep learning model, DeepPathNet, which uses cancer-specific pathway information for both single and multi-omics data analysis. DeePathNet leverages the cutting-edge deep learning technique, Transformer, which is derived from the field of natural language processing, to model complex interactions between pathways from omics data. The computation of self-attention in the Transformer module allows DeePathNet to learn the encoding of pathways to achieve superior predictive performance and interpretability. Techniques such as drop out layers are also integrated into DeePathNet to maximise its generalisability for unseen data. Moreover, DeePathNet supports any number of omics layers and can handle missing values. Using multiple evaluation metrics, we demonstrate that DeePathNet robustly outperforms traditional methods for predicting drug response and cancer type on four publicly available datasets, namely COSMIC Cell Lines, Genomics of Drug Sensitivity in Cancer (GDSC), Cancer Cell Line Encyclopedia (CCLE) and Cancer Therapeutics Response Portal (CTRP). DeePathNet also provides reliable model interpretation, potentially enabling biomarker discoveries at the pathway level. Using the Transformer, DeePathNet is the first method that supports multi-omics data analysis, integrates cancer pathway knowledge into modelling, and provides pathway-level model explanation."

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: "2021-11-23"
date_end: "2021-11-23"
all_day: false

# Schedule page publish date (NOT talk date).
publishDate: "2017-01-01T00:00:00Z"

authors: ["Rebecca C Poulos", "Emanuel Gon√ßalves", "Zhaoxiang Cai", "Syd Barthorpe", "Srikanth S Manda", "Natasha Lucas", "Alexandra Beck", "Daniel Bucio-Noble", "Michael Dausmann", "Caitlin Hall", "Michael Hecker", "Jennifer Koh", "Sadia Mahboob", "Iman Mali", "James Morris", "Laura Richardson", "Akila J Seneviratne", "Erin Sykes", "Frances Thomas", "Sara Valentini", "Steven G Williams", "Yangxiu Wu", "Dylan Xavier", "Karen L MacKenzie", "Peter G Hains", "Brett Tully", "Phillip J Robinson", "Qing Zhong", "Mathew J Garnett", "Roger R Reddel"]
tags: ["Deep Learning", "Machine learning", "Multi-omics", "Pan-cancer", "Drug response", "CRISPR-Cas9"]

# Is this a featured talk? (true/false)
featured: false

---
